util/chplenv/printchplenv.py

```diff
@@ -93,7 +93,7 @@
     ChapelEnv('CHPL_TARGET_BACKEND_CPU', INTERNAL),
     ChapelEnv('CHPL_LOCALE_MODEL', RUNTIME | LAUNCHER | DEFAULT, 'loc'),
     ChapelEnv('  CHPL_GPU', RUNTIME | DEFAULT, 'gpu'),
-    ChapelEnv('  CHPL_GPU_ARCH', INTERNAL),
+    ChapelEnv('  CHPL_GPU_ARCH', DEFAULT),
```

Comment:
This makes `CHPL_GPU_ARCH` print by default; is that an intentional part of this PR? I would imagine at least we'd want it to only print for `CHPL_GPU!=none`.

Reply:
At this point this is remnant from an earlier implementation.